tests/dialects/test_hive.py

```diff
@@ -626,25 +626,25 @@ def test_hive(self):
             },
         )
         self.validate_all(
-            "SELECT * FROM x TABLESAMPLE(1) AS foo",
+            "SELECT * FROM x TABLESAMPLE (1) AS foo",
```

Comment:
why add a space? it's a func isn't it

Reply:
I saw some examples, e.g. in Snowflake, where they used spaces and thought it'd be ok:
```sql
SELECT * FROM testtable TABLESAMPLE (100);  -- Snowflake example
```
Another reason is that before we'd get a space when `method != ""`, which was inconsistent to the no-space function-style formatting in case no method was passed. Examples from our tests before this PR:

```python
SELECT a FROM test TABLESAMPLE BERNOULLI (50) -- Space between BERNOULLI and (50)
SELECT a FROM test TABLESAMPLE(100) -- No space here, inconsistent with above
```

This also works well with DuckDB's syntax for `USING SAMPLE`.